src/index.ts

```diff
@@ -16,16 +16,12 @@ const {
 } = chalk
 
 export interface HelpOptions {
-  all?: boolean;
   maxWidth: number;
   stripAnsi?: boolean;
+  all?: boolean;
```

Comment:
Why move this down?

Reply:
👍 Shuffled in the refactor, I'll put it back.